### PR TITLE
[FEATURE] Operator: Basic Metrics enabled

### PR DESCRIPTION
**Pull request by Pavan-SAP**
Enable basic metrics for the operator:
- controller: workqueue, tenant operation (optional detailed) metrics
- subscription: http handler metrics

Note: For the controller, detailed tenant operation metrics are disabled
by default and can be enabled by setting the environment variable:
`DETAILED_OPERATIONAL_METRICS` to `true`.

Documentation will be published under:
 https://sap.github.io/cap-operator/docs/usage/operator-metrics/

### CODE REVIEW

**Comment by Pavan-SAP** [comment]
Can be tested using:
`ghcr.io/pavan-sap/cap-operator/controller:0.11.0`
&
`ghcr.io/pavan-sap/cap-operator/server:0.11.0`

The metrics server for both is running under port 9090 (and /metrics path by default)

**Review by anirudhprasad-sap — APPROVED** [review]
👍

File: website/content/en/docs/usage/metrics.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Metrics"
```
Comment (skrishnan-sap) [inline website/content/en/docs/usage/metrics.md:2]: Can we change this to "Operator Metrics" to be clear (as this is clubbed under "Usage" where application related processes are documented)?

**Comment by Pavan-SAP** [comment]
[like]  Nayak, Pavan reacted to your message:
________________________________

File: website/content/en/docs/usage/metrics.md (hunk 1 above)
Comment (Pavan-SAP) [reply website/content/en/docs/usage/metrics.md]: Done.. the document is still not up to mark, will work on that separately later.

**Review by skrishnan-sap — APPROVED** [review]
OK - tested with multiple apps.

**Pull request merged**